src/meta/processors/job/RebuildFTJobExecutor.cpp

```diff
@@ -10,14 +10,27 @@ namespace meta {
 
 folly::Future<Status> RebuildFTJobExecutor::executeInternal(HostAddr&& address,
                                                             std::vector<PartitionID>&& parts) {
-  return adminClient_->addTask(cpp2::AdminCmd::REBUILD_FULLTEXT_INDEX,
-                               jobId_,
-                               taskId_++,
-                               space_,
-                               {std::move(address)},
-                               taskParameters_,
-                               std::move(parts),
-                               concurrency_);
+  folly::Promise<Status> pro;
+  auto f = pro.getFuture();
+  adminClient_
```

Comment:
why create another future don't use the return value ?

Reply:
Because it return `Future<StatusOr<bool>`...